### ISSUE TITLE
AfterShipError types are off

### ISSUE THREAD

**Issue by pintojeremy** [AfterShip/aftership-sdk-nodejs#98]
Hi there,

Seems like in `AfterShipError`, the code might be a number in cases of ApiError.

### How to reproduce

```
import {AfterShip} from 'aftership'
import {AftershipError} from "aftership/dist/error/error";

const client = new AfterShip('API_KEY')

await client.tracking.getTracking({slug: 'chronopost-france', tracking_number: '12345'})
    .catch((error: AftershipError) => {
        console.log(typeof error.code);
    })

```

This snippet outputs `number`, code is supposed to be a string.

I don't know what is the best way to fix this for you guys.
Stringify the code could be an answer, union-type is an other one.. 🤔 
I could also see useful an enum with each ApiError you've [described here](https://www.aftership.com/docs/tracking/quickstart/request-errors#http-status-code-summary).

Anyway thanks for your work, hope it helps 🙂

**Comment by panxl6** [comment]
Hi pintojeremy, as AftershipError.code defined in error.ts, the code type is string type. But as Meta.code defined in aftership_response.ts file is number. 
When the request is sent success, axios transfer code into number.  If the request not sent, code field stay string. However, error code defined in api doc is number. [HTTP Status Code](https://www.aftership.com/docs/tracking/quickstart/request-errors#http-status-code-summary)
So i think, the best way is to unify code type to number. I will pull request to make a break change.

**Comment by Bossa573** [comment]
this has been fixed by 8.0.0 (https://www.npmjs.com/package/aftership/v/8.0.0), thank you
